baselines/her/ddpg.py

```diff
@@ -55,12 +55,12 @@ def __init__(self, input_dims, buffer_size, hidden, layers, network_class, polya
             sample_transitions (function) function that samples from the replay buffer
             gamma (float): gamma used for Q learning updates
             reuse (boolean): whether or not the networks should be reused
-            bc_loss: whether or not the behavior cloning loss should be used as an auxilliary loss
-            q_filter: whether or not a filter on the q value update should be used when training with demonstartions
-            num_demo: Number of episodes in to be used in the demonstration buffer
+            bc_loss: whether or not the behavior cloning loss should be used as an auxiliary loss
+            q_filter: whether or not a filter on the q value update should be used when training with demonstrations
+            num_demo: Number of episodes to be used in the demonstration buffer
```

Comment:
I regularly change `in to` into `into`, but this appears to just be an errant `in`...